table/merging_iterator.cc

```diff
@@ -634,9 +656,19 @@ void MergingIterator::SeekImpl(const Slice& target, size_t starting_level,
     for (size_t level = 0; level < starting_level; ++level) {
       if (range_tombstone_iters_[level] &&
           range_tombstone_iters_[level]->Valid()) {
-        assert(static_cast<bool>(active_.count(level)) ==
```

Comment:
range_tombstone_iters_[level] being Valid() does not imply `level` is in `active_` now, since it could be over iterate_upper_bound and was not added to minHeap_.